### DISCUSSION TITLE
Turn off progress bar

### DISCUSSION THREAD

**Discussion by MrSampson** [pymc-labs/pymc-marketing#377] [Q&A] [answered]
When fitting the beta-gamma and the gamma-gamma models, is there a way to suppress the output of the progress bar?

Thanks,
Oliver

**Comment by ricardoV94** [comment, marked as answer]
Have you tried passing `progressbar=False`?

**Reply by MrSampson** [reply]
Thanks for that. I didn't see it in the docs, so I couldn't imagine it would be something quite that simple!
